pkg/platform/api/buildplanner/response/commit.go

```diff
@@ -38,7 +38,7 @@ func processPlanningError(message string, subErrors []*BuildExprError) error {
 			}
 		}
 
-		if se.Type != types.RemediableSolveErrorType && se.Type != types.GenericSolveErrorType {
+		if se.Type != types.RemediableSolveErrorType && se.Type != types.GenericSolveErrorType && se.Type != types.RemediableError {
```

Comment:
Are you sure RemediableSolveErrorType and RemediableError are distinct? I think we only did the solver remediations, so RemediableError might just be the base error, but they're effectively the same.

Reply:
They are different but after some inspection it looks like the `RemediableError` is experimental so I've removed any references to it. https://github.com/ActiveState/TheHomeRepot/blob/master/service/buildplanner/buildplan.graphql#L2388